public/index.html

```diff
@@ -313,7 +313,7 @@
 						<div class="col-lg-6">
 							<!-- Info -->
 							<div class="mb-5">
-								<h1 class="text-white mb-5">It's Time to Tokenize OSS</h1>
+								<h1 class="text-white mb-5">It's Time to Tokenize Open source software</h1>
```

Comment:
Same with this, please uppercase the s.

Ex.:
- **Open Source Software**

--- 
Thank you for contributing <3